_posts/2013-09-26-authentication.markdown

```diff
@@ -10,6 +10,10 @@ title: Authentication Example
 
 Due to the sensitive nature of payroll, all potential integrations must be vetted and approved by Gusto.
 
+When developing an integration, all API calls should be made to api.gusto-demo.com. Once the integration is reviewed by Gusto, we’ll request a callback URI for your production system and issue production API keys at which point all calls should be made to api.gusto.com.
+
+If you'd like to be considered for a future integration partnership, please email partnerships@gusto.com.
```

Comment:
Are we still processing these through partnerships, or do we want to send them to developer-success@?

I would update the email to be a link to make it easier to take action. To make a link, you use brackets and parentheses. The bracketed words are what the user will see. The text in parentheses is the link: 

[partnerships@gusto.com](mailto:partnerships@gusto.com)